packages/cmcd-validator-demo-app/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import Dropdown from 'react-bootstrap/Dropdown';
+import Form from 'react-bootstrap/Form';
+import InputGroup from 'react-bootstrap/InputGroup';
+import Button from 'react-bootstrap/Button'
+import DropdownButton from 'react-bootstrap/DropdownButton';
+
+
+const uris = [
+    { type: 'HLS', value: 'https://dxclj9vp3m44c.cloudfront.net/hls/Costa_Rica_144.m3u8' },
+    { type: 'DASH', value: 'https://livesim.dashif.org/livesim/scte35_2/testpic_2s/Manifest.mpd' },
+    { type: 'DASH', value: 'https://dash.akamaized.net/digitalprimates/fraunhofer/480p_video/heaac_2_0_with_video/Sintel/sintel_480p_heaac2_0.mpd' },
+    { type: 'DASH', value: 'https://cmafref.akamaized.net/cmaf/live-ull/2006350/akambr/out.mpd' },
+    { type: 'DASH', value: 'https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd' },
+  ];
+  
+const SegmentedButtonDropdownsExample = (props) => {

```

Comment:
Change the component name, the name should be equal to the folter thar contains it.